### ISSUE TITLE
How do I install?

### ISSUE THREAD

**Issue by S3j5b0** [garrigue/lablgtk#137]
Hi, I'm trying to install labgltk as dependency for proverif. 
I might be a tad slow, but I really can't figure out how I should install this, from reading the README.

I have downloaded the latest release `2.18.11` and extracted it (im running ubuntu 20.04) . 
I have all the dependencies that I should have, but I really can't figure out what to do?

i've tried to cd into the project and running `dune build` as described in the readme, but it has no effect. 

So, what should I do? what am I missing?

**Comment by drew887** [comment]
Genuinely curious is there a reason you're unable to install this via opam ? I believe that's still the generally preferred method ? :thinking: 

It looks like in the proverif README they mention using the lablgtk README to install if you wanted to do proverif from source. If that's the case it looks like proverif can be installed directly from opam as well if you want, that might be easier unless you have some reason you must install from source instead of via opam :thinking:

**Comment by S3j5b0** [comment]
Hi, I have never worked with ocaml before, and i'm guessing that opam is a package manager for ocaml , but I have absolutely no idea what it is.
So, I just have no idea how to use opam, googling stuff like "opam install proverif" gives nothing, same for this package .
Heh, so I guess i'm just a little bit out of bounds

**Comment by drew887** [comment]
Yeah opam is a package manager for ocaml, im surprised your google search turned up nothing considering that `opam install proverif` is verbatim on the main page of the proverif site, https://prosecco.gforge.inria.fr/personal/bblanche/proverif/ as well as the main readme for proverif, but google be weird like that sometimes. 

As for how to install opam that's best left to opam's documentation for your operating system of choice but I'll copy paste the instructions from the proverif readme here for once you have it installed
```
******* Installation through OPAM

OPAM can be downloaded from:
	https://opam.ocaml.org
After installing OPAM, do:
      opam install proverif
      opam depext proverif
(The second line should install graphviz if you do not already have it.
You may also choose to install graphviz manually if needed.)

```

**Comment by drew887** [comment]
Either way though @S3j5b0 do mind please closing this issue as its not so much an issue with lablgtk as much as this is just proverif/opam installation
